docs/reference/ilm/ilm-actions.asciidoc

```diff
@@ -63,4 +67,7 @@ include::actions/ilm-set-priority.asciidoc[]
 include::actions/ilm-shrink.asciidoc[]
 include::actions/ilm-unfollow.asciidoc[]
 include::actions/ilm-wait-for-snapshot.asciidoc[]
+
+ifdef::permanently-unreleased-branch[]
 include::actions/ilm-rollup.asciidoc[]
+endif::[]
```

Comment:
Hey @jrodewig, mind letting me know if I did this right? thanks!

Reply:
This looks correct to me!

Reply:
thanks!